sql/2022/javascript/requests_2022.sql

```diff
@@ -0,0 +1,15 @@
+#standardSQL
+# Sum of JS requests per page (2022)
+SELECT
+  percentile,
+  _TABLE_SUFFIX AS client,
+  APPROX_QUANTILES(reqJS, 1000)[OFFSET(percentile * 10)] AS js_requests
+FROM
+  `httparchive.summary_pages.2022_06_01_*`,
```

Comment:
Similar comment as the byte distribution over time, I'd suggest combining them all with `UNION ALL`.